src/bigquery.ts

```diff
@@ -80,15 +80,18 @@ export type PagedRequest<P> = P & {
   maxApiCalls?: number;
 };
 
+export type QueryResultsResponse = bigquery.IGetQueryResultsResponse &
```

Comment:
this syntax feels absolutely wild to me, a concrete type made from ANDing two interfaces? Should it be IQueryResultsResponse?
Does this blow up if you have fields with equal names but varying types?

Reply:
Re Alvaro, I think the OR is probably a better choice